File: Java/X - XIII Programming Exercise (Recursive Power Method).java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.print("Full Name: ");
        String name = scan.nextLine();

        System.out.print("Enter the base: ");
        double base = scan.nextDouble();

        System.out.print("Enter the exponent: ");
        int exponent = scan.nextInt();

        double result = power(base, exponent);

        System.out.println(base + " raised to the power of " + exponent + " is " + result);

        scan.close();
    }

    public static double power(double base, int exponent) {
        double result = 1.0;
        for (int i = 0; i < exponent; i++) {
            result *= base;
        }
        return result;
    }
}

File: Python/X - XIII Programming Exercise (Recursive Power Method).py
str_name = str(input("Full Name: "))

def power(base, exponent):
    if exponent == 0:
        return 1
    else:
        return base * power(base, exponent - 1)

base = float(input("Enter the base: "))
exponent = int(input("Enter the exponent: "))

result = power(base, exponent)

print(base, "raised to the power of", exponent, "is", result)
